### ISSUE TITLE
Figure out how to store objects

### ISSUE THREAD

**Issue by taragor** [taragor/AirlockControlSE#5]
As said above; so not all airlocks have to be fetched at every single programm runtime.

**Comment by bluePlatinum** [comment]
Your issues #5 and #6 contradict themselves

**Comment by taragor** [comment]
no; all airlocks have to be fetched at "boot", ie. after server start or recompile, and then should be stored for further use, to optimize runtime

**Comment by bluePlatinum** [comment]
ah i see where your coming from; will do

**Comment by bluePlatinum** [comment]
"The Save method is called whenever the game is saved, or right before you recompile your script. Here you can store any data you need to persist between play sessions. "
